test/verify/machineslib.py

```diff
@@ -1162,8 +1162,32 @@ def installWithErrorTest(dialog):
                                          start_vm=True))
         # End of tests for import existing disk as installation option
 
-        # test PXE Source
         if self.provider == "libvirt-dbus":
+            self.machine.execute("qemu-img create -f qcow2 /var/lib/libvirt/vmTestCustomDestination.qcow2 128M")
+            self.machine.execute("virsh pool-refresh libvirt")
+
+            # Check choosing existing volume as destination storage
```

Comment:
The following two tests are exposed with the virsh provider for users, thus testing them should not be limited to libvirt-dbus being installed.